pkgs/data/fonts/joypixels/default.nix

```diff
@@ -1,24 +1,95 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, acceptLicense ? false }:
```

Comment:
Are you sure a font warrants a configuration option? nix-env users, and everyone else, would be able to use a simple overlay, no?

Reply:
`config` values are used very freely in nixpkgs, since they don't cost anything. And setting a `config` value is pretty easy on both NixOS and with `nix-env`, much easier than an overlay. An overlay for `nix-env` to do such an override would be such a file in ~/.config/nixpkgs/overlays/joypixel-accept
```nix
self: super: {
  joypixels = super.joypixels.override { acceptLicense = true; };
}
```

In comparison, with `config`, it's just this in ~/.config/nixpkgs/config.nix:
```nix
{
  joypixels.acceptLicense = true;
}
```

And it's similarly easy on NixOS

Reply:
Applied the suggestion 👍 
The way I did it actually allows *both* specifying the config option or overriding the argument so now *everybody*'s happy : )